plugins/rest/komand_rest/actions/delete/__init__.py

```diff
@@ -1,2 +1,2 @@
-# GENERATED BY KOMAND SDK - DO NOT EDIT
-from .action import Delete
+# GENERATED BY INSIGHT-PLUGIN - DO NOT EDIT
+from .action import Delete
```

Comment:
```suggestion
from .action import Delete

```

Reply:
The updated Insight tooling should stop this from happening in the future, be sure to update your tooling to the newest version.